metrics/default-mobile-app-metrics.md

```diff
@@ -0,0 +1,159 @@
+# Abstract
+
+This document contains the proposal for gathering some default metrics of mobile apps and making them available to developers.
+
+## Goals
+
+### Goals
+
+- Provide mobile app developers a service that can receive some basic default metrics, namely sdk version and app version
+
+- Provide developers an interface to query and visualize the gathered data in order to make informed decisions
+
+### Non Goals
+
+- Identify individual devices or users in metrics events (bonus)
+
+- Registering of user-defined metrics and logs (not prioritized; just keep in mind)
+
+- Provide mobile app developers with an easy to use SDK to connect easily and securely to the metrics service (separate epic/proposal)
+
+## Terms
+
+For a list of terms used in this document, see [Glossary](./glossary.md).
+
+## Problem Description
+
+As mobile apps are developed, improved and iterated on, different versions of those apps can be used on many different devices. 
+Metrics data becomes invaluable for developers to observe the landscape of application installations and make sure further 
+development and deployment of backend services will not affect users.
+
+## Proposed Solution
+
+A Metrics Service will be available from the Service Catalog, which can be provisioned by developers for use with their mobile applications.
+
+The following diagram illustrates the components:
+
+![service diagram](./default-mobile-metrics-system-landscape.png)
+
+### Mobile App Client
+
+A native, platform-specific SDK for targeting the Metrics Service will be available to developers.
+
+Upon installation into a project and providing configuration for targeting the Metrics Service the SDK will automatically 
+send the default metrics information for the application with a unique ID to the metrics service.
+This information will be sent to the service every time the application starts.  
+
+Please note that, the client side is not part of this proposal.
+
+### Mobile Metrics Service
+
+A thin HTTP-based service will be present to receive metrics data from the mobile applications and then to forward to the storage service.
+
+At a PoC level, the endpoints available from the Mobile Metrics Service should be:
+
+`POST /default-metrics`
```

Comment:
Don't know how applicable this is but I have a suggestion. Would it make sense to version this API? e.g. something like `/api/1.0/default-metrics`?

I feel like the metrics will change a lot over time. Just a suggestion.

Reply:
@darahayes Good point. 
However, I think for and MVP of a metrics service this may be adding unnecessary complexity to the service and SDKs.

Reply:
@david-martin I can see an "API versioning" card in product Trello board (https://trello.com/c/71UiKc1F/12-as-a-developer-i-want-to-be-able-to-version-my-apis-and-manage-different-versions-for-different-sets-of-client-apps-i-will-need). But it is about versioning custom user APIs. I don't see anything about versioning SDK API versioning. You might want to talk about this with @johnfriz for future.

@darahayes nice catch!